src/main/java/io/pravega/connectors/flink/sink/PravegaEventWriter.java

```diff
@@ -92,6 +93,15 @@
     // The writer id
     private final String writerId;
 
+    // flag to enable/disable metrics
+    private final boolean enableMetrics;
```

Comment:
Normally `enableMetrics` flag is to mark if we need to connect to the internal client metric system, which may have some impact on the client behavior and performance. As the current counter is done on connector side, we can by default enable metrics and remove the flag.